service/worker/diagnostics/workflow.go

```diff
@@ -33,8 +33,9 @@ import (
 )
 
 const (
-	diagnosticsWorkflow = "diagnostics-workflow"
-	tasklist            = "wf-diagnostics"
+	diagnosticsWorkflow    = "diagnostics-workflow"
+	tasklist               = "wf-diagnostics"
```

Comment:
nit: I'd name it maybe "diagnostics-tasklist" or something similar for consistency in names

Reply:
done